unbxdAutosuggest.js

```diff
@@ -496,11 +512,67 @@ var unbxdAutoSuggestFunction = function($,Handlebars,undefined){
 			}
 		}
 		,addToAnalytics:function(type,obj){
-			if("Unbxd" in window && "track" in window.Unbxd && typeof window.Unbxd.track == "function"){
-				this.log("Pushing data to analytics",type,obj);
-				Unbxd.track( type, obj );
-			}
+		  if("Unbxd" in window && "track" in window.Unbxd && typeof window.Unbxd.track == "function"){
+		    this.log("Pushing data to analytics",type,obj);
+		    Unbxd.track( type, obj );
+		  }
+		  if(type !== 'search') return;
+
+		  if('classical' in this.options.integrations){
+		    this.trackclassical(type, obj);
+		  }
+		  if('universal' in this.options.integrations){
+		    this.trackuniversal(type, obj);
+		  }
 		}
+	  ,getEventAction: function(autosuggestType){
+	    var types = {
+	      'IN_FIELD': 'Scope_Click',
+	      'POPULAR_PRODUCTS': 'Pop_Click',
+	      'KEYWORD_SUGGESTION': 'TQ_Click',
+	      'TOP_SEARCH_QUERIES': 'TQ_Click'
+	    }
+	    return types[autosuggestType];
+	  }
+	  ,getEventLabel: function(autosuggest){
+	    var params = autosuggest.autosuggestParams;
+	    var value = params.autosuggest_suggestion;
+	    var index = params.unbxdprank;
+	    var filter = params.field_name && params.field_value ?
+		params.field_name + ':' + params.field_value : undefined;
+	    var types = {
+	      'IN_FIELD': value + (filter ? '&filter='+filter : '' )+ '-' + index,
+	      'POPULAR_PRODUCTS': value + '-' + index,
+	      'KEYWORD_SUGGESTION': value + '-' + index,
+	      'TOP_SEARCH_QUERIES': value + '-' + index
+	    };
+	    return types[params.autosuggest_type];
+	  }
+	  ,trackclassical: function(type, obj){
+	    var key = this.options.integrations['classical'],
+		eventAction = this.getEventAction(obj.autosuggestParams.autosuggest_type),
+		eventLabel = this.getEventLabel(obj),
+		value = 1;
+	    if(key){
+	      if(key && key === true){
```

Comment:
instead of 2 if conditions, can we write this as **if( typeof key !== 'undefined' && key === true )** for more readability ?